foladipo/docs-cabinet-cp2
# Chore/146970561/finish backend tests

#### What does this PR do?

Adds more backend tests to properly cover the backend's functionality. It also implements various feedback gotten from CP2 defence thus far.

#### Description of task to be completed

Complete backend tests.

#### How should this be manually tested?

- Follow the instructions stated in this project's README to set up your local instance of this app.

- Open a terminal/command prompt and enter `npm run test:server` to run the tests.

- Enter `npm run test:nyc` to see the test coverage in your terminal.

- As an alternative, visit [this link](https://coveralls.io/github/andela-foladipo/docs-cabinet-cp2?branch=chore%2F147025811%2Fsetup-frontend-dev-environment) to see the test coverage for the backend.

#### Any background context you want to provide?

N/A

#### What are the relevant Pivotal Tracker stories?

146970561

#### Screenshots
<img width="1242" alt="screen shot 2017-07-02 at 12 40 38 am" src="https://user-images.githubusercontent.com/26963575/27766161-24cca828-5ebf-11e7-8560-2b491c395f24.png">
